lib/ruby_units/configuration.rb

```diff
@@ -24,17 +24,18 @@ def self.configure
 
   # holds actual configuration values for RubyUnits
   class Configuration
-    # used to separate the scalar from the unit when generating output.
-    # set to nil to prevent adding a space to the string representation of a unit
-    # separators other than ' ' and '' may work, but you may encounter problems
+    # Used to separate the scalar from the unit when generating output. A value
```

Comment:
Initially I thought the problem might be related to the optional separator feature. That was wrong, but I noticed that the documentation was also wrong, so I fixed it.